src/Native/SymSgdNative/SymSgdNative.cpp

```diff
@@ -240,7 +238,6 @@ void InitializeState(int totalNumInstances, int* instSizes, int** instIndices, f
         TuneNumLocIter(numLocIter, totalNumInstances, instSizes, numThreads);
 
     state->WeightScaling = 1.0f;
-#if defined(USE_OMP)  
```

Comment:
>#if defined(USE_OMP)  [](start = 0, length = 21)

why it's going away?

Reply:
Maybe we should just define `USE_OMP` in the CMake files instead?

Reply:
I'm fine with adding back and adding the USE_OMP to  CMakefile file.

---
In reply to: [262995598](https://github.com/dotnet/machinelearning/pull/2867#discussion_r262995598) [](ancestors = 262995598)